### PR TITLE
[release-1.7] Update Performance and Scalability Page for 1.7

### PR DESCRIPTION
**Pull request by carolynhu**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ x ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

https://github.com/istio/istio.io/issues/7271

### CODE REVIEW

**Comment by carolynhu** [comment]
close this pr, since this branch is messed up

File: content/en/docs/ops/deployment/performance-and-scalability/index.md [hunk 1]
```diff
@@ -108,32 +108,48 @@ is busy handling the request, the worker won't start handling the next request
...
 >}}
 
 - `baseline` Client pod directly calls the server pod, no sidecars are present.
-- `none-both` Istio proxy with no Istio specific filters configured.
-- `telemetryv2-both` client and server sidecars are present with telemetry v2 `nullvm` configured by default.
-- `mixer-both` Client and server sidecars are present with mixer configured.
+- `none_both` Istio proxy with no Istio specific filters configured.
+- `v2-stats-wasm_both` Client and server sidecars are present with telemetry v2 `v8` configured.
+- `v2-stats-nullvm_both` Client and server sidecars are present with telemetry v2 `nullvm` configured by default.
+- `v2-sd-full-nullvm_both` Export stackdriver metrics, access logs and edges with telemetry v2 `nullvm` configured.
```
Comment (bianpengyuan) [inline content/en/docs/ops/deployment/performance-and-scalability/index.md:151]: No need to include sd in the result.

File: content/en/docs/ops/deployment/performance-and-scalability/index.md [hunk 2]
```diff
@@ -98,8 +98,8 @@ request rate doesn't affect the memory consumption.
 ### Latency
 
 Since Istio injects a sidecar proxy on the data path, latency is an important
-consideration. Istio adds an authentication and a Mixer filter to the proxy. Every
-additional filter adds to the path length inside the proxy and affects latency.
+consideration. Istio adds an authentication and a telemetry filter and a metadata exchange filter
```
Comment (bianpengyuan) [inline content/en/docs/ops/deployment/performance-and-scalability/index.md:101]:
```suggestion
consideration. Istio adds an authentication filter, a telemetry filter, and a metadata exchange filter
```

File: content/en/docs/ops/deployment/performance-and-scalability/index.md [hunk 3]
```diff
@@ -108,32 +108,48 @@ is busy handling the request, the worker won't start handling the next request
 immediately. This process adds to the queue wait time of the next request and affects
 average and tail latencies. The actual tail latency depends on the traffic pattern.
 
+Note: in Istio release 1.7, we are introducing a new way of measuring performance by enabling `jitter` in
+the load generator. It helps by modelling random traffic from the client side when using connection pools. In the next
```
Comment (frankbu) [inline content/en/docs/ops/deployment/performance-and-scalability/index.md:112]:
```suggestion
the load generator. It helps by modeling random traffic from the client side when using connection pools. In the next
```

File: content/en/docs/ops/deployment/performance-and-scalability/index.md (hunk 1 above)
Comment (frankbu) [reply content/en/docs/ops/deployment/performance-and-scalability/index.md:151]: If you don't remove this line, it has a lint spelling error:
```suggestion
- `v2-sd-full-nullvm_both` Export Stackdriver metrics, access logs and edges with telemetry v2 `nullvm` configured.
```

**Review by frankbu — COMMENTED** [review]
A couple of lint errors

File: content/en/docs/ops/deployment/performance-and-scalability/index.md (hunk 1 above)
Comment (carolynhu) [reply content/en/docs/ops/deployment/performance-and-scalability/index.md:151]: Offline discussed, will keep the stackdriver numbers.

**Comment by carolynhu** [comment]
Put on hold, there are others numbers and descriptions need to be updated to the latest status.

File: content/en/docs/ops/deployment/performance-and-scalability/index.md [hunk 4]
```diff
@@ -77,22 +76,18 @@ Data plane performance depends on many factors, for example:
...
 ### CPU and memory
 
 Since the sidecar proxy performs additional work on the data path, it consumes CPU
-and memory. As of Istio 1.1, a proxy consumes about 0.6 vCPU per 1000
+and memory. As of Istio 1.7, a proxy consumes about 0.6 vCPU per 1000
 requests per second.
 
 The memory consumption of the proxy depends on the total configuration state the proxy holds.
-A large number of listeners, clusters, and routes can increase memory usage.
-Istio 1.1 introduced namespace isolation to limit the scope of the configuration sent
```
Comment (bianpengyuan) [inline content/en/docs/ops/deployment/performance-and-scalability/index.md:92]: Let's keep this line as it is.

File: content/en/docs/ops/deployment/performance-and-scalability/index.md [hunk 5]
```diff
@@ -77,22 +76,18 @@ Data plane performance depends on many factors, for example:
...
-- Number and types of proxy filters, specifically Mixer filter.
+- Number and types of proxy filters, specifically telemetry v2 related filters.
 
 The latency, throughput, and the proxies' CPU and memory consumption are measured as a function of said factors.
 
 ### CPU and memory
 
 Since the sidecar proxy performs additional work on the data path, it consumes CPU
-and memory. As of Istio 1.1, a proxy consumes about 0.6 vCPU per 1000
+and memory. As of Istio 1.7, a proxy consumes about 0.6 vCPU per 1000
```
Comment (bianpengyuan) [inline content/en/docs/ops/deployment/performance-and-scalability/index.md:86]: Is this number still up to dated?

Comment (carolynhu) [reply content/en/docs/ops/deployment/performance-and-scalability/index.md]: updated to the latest.

**Comment by carolynhu** [comment]
synced with @howardjohn, we will have some detailed update and explanations about Control plane perf metrics later on. So here keep as what since from istio 1.1 is.

**Review by bianpengyuan — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

**Pull request merged**